### PR TITLE
Chore/test/student exam import dialog

### PR DESCRIPTION
**Pull request by ivanchimeno**
### Checklist
- [x] Client: I followed the [coding and design guidelines](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client.html).
- [x] Client: I added multiple integration tests (Jest) related to the features (with a high test coverage)

### Motivation and Context
Code coverage improvement for exam components.

### Description

Added client tests for  `student-exam-import-dialog.component.ts`

### Test Coverage

student-exam-import-dialog.component.ts 91.36%

### CODE REVIEW

File: src/test/javascript/spec/component/exam/manage/student-exams/student-exam-import-dialog.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+import * as sinonChai from 'sinon-chai';
+import { Course } from 'app/entities/course.model';
+import * as sinon from 'sinon';
+import { Exam } from 'app/entities/exam.model';
+import { StudentsExamImportDialogComponent } from 'app/exam/manage/students/students-exam-import-dialog/students-exam-import-dialog.component';
+import { TranslatePipe } from '@ngx-translate/core';
+import { FormsModule } from '@angular/forms';
+import { JhiTranslateDirective } from 'ng-jhipster';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { DebugElement } from '@angular/core';
```
Comment (FrankeLukas) [inline src/test/javascript/spec/component/exam/manage/student-exams/student-exam-import-dialog.component.spec.ts:12]: This variable is unused.

File: src/test/javascript/spec/component/exam/manage/student-exams/student-exam-import-dialog.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+import { ExamManagementService } from 'app/exam/manage/exam-management.service';
+import { MockComponent, MockDirective, MockPipe, MockProvider } from 'ng-mocks';
+import { JhiAlertService } from 'ng-jhipster';
+import { FaIconComponent } from '@fortawesome/angular-fontawesome';
+import { HelpIconComponent } from 'app/shared/components/help-icon.component';
+import { AlertComponent } from 'app/shared/alert/alert.component';
+import { AlertErrorComponent } from 'app/shared/alert/alert-error.component';
+import { By } from '@angular/platform-browser';
```
Comment (FrankeLukas) [inline src/test/javascript/spec/component/exam/manage/student-exams/student-exam-import-dialog.component.spec.ts:20]: Same here.

**Review by FrankeLukas — APPROVED** [review]
Lgtm otherwise.

**Review by FrankeLukas — APPROVED** [review]

**Review by FuchsDominik — APPROVED** [review]
Looks good :)

**Review by krusche — APPROVED** [review]

**Pull request merged**